Web/BLL/Services/Speaker.cs

```diff
@@ -10,15 +10,11 @@ namespace RockFood.Services
 {
     public static class Speaker
     {
-        private static readonly Logger _logger = new Logger();
         public static void Output(string message, string messageAuthor = "System")
         {
             Console.WriteLine("{0}: {1}", messageAuthor, message);
```

Comment:
using "Console.WriteLine" show me that it is console app
rechecck please
add remove all "Console.WriteLine"